src/browser/modules/Frame/FrameAside.jsx

```diff
@@ -21,14 +21,15 @@
 import React, { Component } from 'react'
 import { H3 } from 'browser-components/headers'
 import { Lead } from 'browser-components/Text'
+import { DARK_THEME, LIGHT_THEME } from 'shared/modules/settings/settingsDuck'
 
 class FrameAside extends Component {
   render() {
     const { subtitle } = this.props
     let { title } = this.props
 
     // Use logo as title if title is only Neo4j
-    if (title === 'Neo4j') {
+    if ((title === 'Neo4j', LIGHT_THEME)) {
```

Comment:
Using , here is a bit confusing as to what is meant. I'd say use && or || instead